### DISCUSSION TITLE
HomeKit2: Is it safe to use the new "Homekit Architecture Update" that became available in iOS 16.4?

### DISCUSSION THREAD

**Discussion by nanosonde** [koush/scrypted#669] [General]
There is a German news article here: https://www.heise.de/news/HomeKit-2-Apple-wagt-neuen-Anlauf-mit-iOS-16-4-8149933.html

Basically it says that all Apple iOS devices have to be on at least 16.4 and official third-party Homekit devices shall also offer a new update to be compatible with HomeKit2. Older iOS version and devices without updates will loose connectivity and cannot take part anymore.

Does anybody have technical details? Will scrypted still work?

**Comment by miwagner1** [comment]
Basically your Apple devices will just talk to the HomeKit hubs instead of each HomeKit accessory individually. HomeKit accessories don’t need any updates, they will work just fine if not better as they will no longer be potentially overloaded with connections.

Once you update to the new architecture via the Home app, your iOS devices need to be on 16.4 or will most likely have some issues.

Since this update is transparent to the HomeKit accessories, all HomeKit accessories including scrypted will continue to work and should work even better than before with faster response times.

Side note: your HomeKit network will now depend on the HomeKit hubs, I would recommend you to have more than 1.
